modules/tinymce/src/themes/silver/test/ts/webdriver/editor/DisabledNestedMenuItemTest.ts

```diff
@@ -0,0 +1,72 @@
+import { Keys, Mouse, RealKeys, UiFinder, Waiter } from '@ephox/agar';
+import { afterEach, describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { SugarBody } from '@ephox/sugar';
+
+import { Editor } from 'tinymce/core/api/PublicApi';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('webdriver.tinymce.themes.silver.editor.menubar.DisabledNestedMenuItemTest', () => {
+
+  const preferencesMenuItemSelector = '[title="Preferences"]';
+  const codeMenuItemSelector = '[role="menuitem"]:contains("Code")';
+
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    menu: {
+      custom: { title: 'Code', items: 'about restart preferences' }
+    },
+    menubar: 'custom',
+    setup: (editor: Editor) => {
+      editor.ui.registry.addMenuItem('about', {
+        text: 'About',
+      });
+
+      editor.ui.registry.addMenuItem('restart', {
+        text: 'Restart',
+      });
+
+      editor.ui.registry.addNestedMenuItem('preferences', {
+        text: 'Preferences',
+        disabled: true,
+        getSubmenuItems: () => {
+          return [
+            {
+              type: 'menuitem',
+              text: 'Settings',
+            }
+          ];
+        }
```

Comment:
```suggestion
        getSubmenuItems: () => [{
          type: 'menuitem',
          text: 'Settings',
        }]
```
This can be slimmed down by using shorthand syntax.